blueocean-web/src/main/java/io/jenkins/blueocean/dev/RunBundleWatches.java

```diff
@@ -370,6 +375,21 @@ public void run() {
         }
     }
 
+    private static boolean isBlueoceanPlugin(PluginManager pluginManager, PluginWrapper p) {
+        if (p == null) {
+            return false;
+        }
+        if (p.getShortName().contains("blueocean")) {
```

Comment:
that means that third party plugins would not be bundled if they are not contain `blueocean` 

That would have the consequence that our company plugin would not bundle, not sure if that is what we want.

Reply:
@scherler it inspects all dependencies. If your plugin does not have a dependency on any `blueocean` plugin, then it won't get automatically bundled. If it does - e.g. `blueocean-core-js`, then it will be considered a "blueocean" plugin

Reply:
that said, i think this may be excluding `jdl`, let me see if there's a better way

Reply:
Might also want to consider whether we need to check all transitive deps for a blueocean-plugin dependency. Not sure about specifics of @scherler 's setup, but I seem to recall that there are two projects that depend on a "common" project, and that depends on blueocean? If so if we aren't checking transitive deps, then auto-build would work for "common" but not for the other two?

Reply:
It's already checking transitive dependencies. But it's a bit of a hackish way, I need to find a better way.